### PR TITLE
Refactor/quick facts admin 284

### PR DESCRIPTION
**Pull request by dabick14**
## Description

This PR refactors the quick facts queries to use the current aggregates structure of the database

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?
This has been tested on my localhost

### CODE REVIEW

File: api/src/schema/directory-quick-facts.graphql [hunk 1]
```diff
@@ -112,17 +95,19 @@ extend type Bacenta {
...
-      MATCH (this)-[:HAS]->(fellowship) WHERE fellowship:Fellowship OR fellowship:ClosedFellowship
-      MATCH (fellowship)-[:HAS_HISTORY]-(:ServiceLog)-[:HAS_SERVICE]->(record:ServiceRecord)
-      MATCH (record) WHERE NOT (record:NoService)
-      MATCH (record)-[:SERVICE_HELD_ON]->(date:TimeGraph)
-      MATCH (date)
-      WHERE date.date > date() - duration({days: $days})
-      WITH sum(record.attendance) AS attendance, sum(record.income) as income, this, date(date.date).week as weeknumber
+      MATCH (this)
+      WITH date() as today, this
+      WITH date(today) - duration({days: 200}) AS startDate, this
```
Comment (jaedag) [inline api/src/schema/directory-quick-facts.graphql:100]: How come you took away the days argument?

**Review by jaedag — APPROVED** [review]

File: api/src/schema/directory-quick-facts.graphql (hunk 1 above)
Comment (dabick14) [reply api/src/schema/directory-quick-facts.graphql:100]: This is an over sight

**Comment by jaedag** [comment]
3 merge commits in the middle because?

**Comment by dabick14** [comment]
> 3 merge commits in the middle because?

3 things were pushed to deploy in the interim

**Comment by jaedag** [comment]
> > 3 merge commits in the middle because?
> 
> 3 things were pushed to deploy in the interim

So? 

Someone told you that everytime something is pushed to deploy you have to merge it?

**Review by jaedag — APPROVED** [review]

**Pull request merged**